core/src/test/java/io/grpc/PickFirstLoadBalancerTest.java

```diff
@@ -71,6 +73,7 @@
 
   private static final Attributes.Key<String> FOO = Attributes.Key.of("foo");
   private Attributes affinity = Attributes.newBuilder().set(FOO, "bar").build();
+  private static final PickSubchannelArgs EMPTY_ARGS = mock(PickSubchannelArgs.class);
```

Comment:
We don't want to set an example for users and let them think they can mock `PickSubchannelArgs` whenever we want. We'd better put a comment here says because this LoadBalancer doesn't use any of the arg fields, as verified in `tearDown()`, it's safe to use a mock without any stubbing. `unusedArgs` seems to be a more accurate name. I would also make it non-static, so that if any tests violates this precondition, we will know exactly which they are.